test/server/storage/safestorage.spec.js

```diff
@@ -1918,7 +1933,12 @@ describe('SafeStorage', function () {
                     throw new Error('Should have failed!');
                 })
                 .catch(function (err) {
-                    expect(err.message).to.contain('Invalid argument, data.projectName failed regexp: ');
+                    try {
+                        expect(err.message).to.contain('Invalid argument, data.projectName failed regexp: ');
```

Comment:
Was adding these to track down why the tests just failed with no error.. Reformatted the file when saving and then it started to complain about indent.. 
I broke up the var statements to fix those and while I did that I added const instead of var.. Down the rabbit-hole in other words.